### DISCUSSION TITLE
24h graphs instead of since midnight.

### DISCUSSION THREAD

**Discussion by lpaseen** [poblabs/weewx-belchertown#889] [Q&A]
I'm trying to get graphs for the last 24h instead of since midnight but can't find any setting for it.
Is it possible?

**Comment by poblabs** [comment]
Yes, try the [day_ago_to_now](https://github.com/poblabs/weewx-belchertown/wiki/Belchertown-Charts-Documentation#time_length) 

https://github.com/poblabs/weewx-belchertown/wiki/Belchertown-Charts-Documentation#start_at_midnight

**Reply by poblabs** [reply]
Or I guess better yet would be hour_ago_to_now, and specify the time_ago as 24

> hour_ago_to_now = As of skin version 1.2. Use with time_ago to specify the number of hours ago to chart up until the current time. Example: time_ago = 5 would chart 5 hours ago to now.

**Reply by lpaseenba** [reply]
At first I couldn't get it to work but then I realised "time_length" and after changing that from "today" to "90000"  it now works
thanks
